src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/transport/PerformanceAnalyzerTransportRequestHandler.java

```diff
@@ -65,6 +65,7 @@ TransportChannel getChannel(T request, TransportChannel channel, Task task) {
     private TransportChannel getShardBulkChannel(T request, TransportChannel channel, Task task) {
         String className = request.getClass().getName();
         boolean bPrimary = false;
+        boolean logOnce = false;
```

Comment:
Let's make it a class member and not a method local.

Reply:
Done.